main.py

```diff
@@ -0,0 +1,35 @@
+from funk import searchingGame, mathematicOper
+import time
```

Comment:
It is common to sort imports in Python files. There are multiple styles (profiles) people use for sorting imports. Personally I prefer [google](https://pycqa.github.io/isort/docs/configuration/profiles.html) style profile.

To automatically sort imports for python files, people use [isort](https://pycqa.github.io/isort/index.html).